transport_test.go

```diff
@@ -50,13 +49,83 @@ func TestTransport(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestNewTransport(t *testing.T) {
+	config := &Config{
+		ConsumerKey:    "consumer_key",
+		ConsumerSecret: "consumer_secret",
+		Noncer:         &fixedNoncer{"some_nonce"},
+	}
+	auther := &DefaultAuther{
+		config: config,
+		clock:  &fixedClock{time.Unix(123456789, 0)},
+	}
+	source := StaticTokenSource(NewToken("access_token", "some_secret"))
+
+	type testCase struct {
+		description string
+		base        http.RoundTripper
+		source      TokenSource
+		auther      Auther
+		errMsg      string
+	}
+	for _, tc := range []testCase{
+		{
+			description: "default base transport",
+			base:        nil,
+			source:      source,
+			auther:      auther,
+			errMsg:      "",
+		},
+		{
+			description: "custom base transport",
+			base:        nil,
```

Comment:
need to use custom transport ?

Reply:
because there is `TestTransport_customBaseTransport`. 
most of the test cases follow existing ones below

Reply:
currently `custom base transport` is the same as `default base transport`

Reply:
fixed in 30c7438e6a01be134ceb4dad553cdbb0d2e81acd